### PR TITLE
[IMPROVEMENT][THEME] Use the new colors for the dark theme

### PR DESCRIPTION
**Pull request by Sameesunkaria**
This PR also includes some minor improvements.

simulator screen shot - iphone x - 2018-06-20 at 12 32 03

### CODE REVIEW

File: Rocket.Chat.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -687,7 +688,7 @@
 		99923773204B3BD800C2D15F /* UploadAvatarRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 99923772204B3BD800C2D15F /* UploadAvatarRequest.swift */; };
 		999483EB20644CC4004F61CA /* WebBrowserViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 999483EA20644CC4004F61CA /* WebBrowserViewModel.swift */; };
 		999F25B5207EEB0C002E0F68 /* FileModelMapping.swift in Sources */ = {isa = PBXBuildFile; fileRef = 999F25B4207EEB0C002E0F68 /* FileModelMapping.swift */; };
-		999F25B7207EEE70002E0F68 /* BuildFile in Sources */ = {isa = PBXBuildFile; };
+		999F25B7207EEE70002E0F68 /* (null) in Sources */ = {isa = PBXBuildFile; };
```
Comment (rafaelks) [inline Rocket.Chat.xcodeproj/project.pbxproj]: 👻

File: Rocket.Chat.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -4317,7 +4321,7 @@
 				80A2F39220057AD0005D2DCA /* EmojiAutocompleteCell.swift in Sources */,
 				41DF76E31D2C50710028DBF8 /* AppDelegate.swift in Sources */,
 				D18675EC1F716A0D00406FB4 /* LoginRequest.swift in Sources */,
-				999F25B7207EEE70002E0F68 /* BuildFile in Sources */,
+				999F25B7207EEE70002E0F68 /* (null) in Sources */,
 				806728FF20079734009FE94D /* DeleteMessageRequest.swift in Sources */,
 				99DBB872208FF6FE00382DB2 /* SearchMessagesRequest.swift in Sources */,
 				80054CF11FD9505A00F5ECF9 /* SendMessageRequest.swift in Sources */,
```
Comment (rafaelks) [inline Rocket.Chat.xcodeproj/project.pbxproj]: Are these changes necessary?

File: Rocket.Chat/External/RCEmojiKit/Views/EmojiPicker/EmojiPicker.swift [hunk 3]
```diff
@@ -351,14 +351,13 @@ extension EmojiPicker {
         guard isPopover else { return theme }
         let popoverTheme = Theme(
             backgroundColor: theme.focusedBackground,
-            titleText: theme.titleText,
+            focusedBackground: theme.focusedBackground, auxiliaryBackground: theme.auxiliaryBackground, titleText: theme.titleText,
```
Comment (rafaelks) [inline Rocket.Chat/External/RCEmojiKit/Views/EmojiPicker/EmojiPicker.swift]: Break lines here?

File: Rocket.Chat/External/RCEmojiKit/Views/Reaction/ReactorListView.swift [hunk 4]
```diff
@@ -163,14 +163,14 @@ extension ReactorListView {
         guard isPopover else { return theme }
         let popoverTheme = Theme(
             backgroundColor: theme.focusedBackground,
-            titleText: theme.titleText,
+            focusedBackground: theme.focusedBackground,
+            auxiliaryBackground: theme.auxiliaryBackground, titleText: theme.titleText,
```
Comment (rafaelks) [inline Rocket.Chat/External/RCEmojiKit/Views/Reaction/ReactorListView.swift]: break likes?

File: Rocket.Chat.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Sameesunkaria) [reply Rocket.Chat.xcodeproj/project.pbxproj]: Not sure what that is, but I added a file so need some changes in the `project.pbxproj`

Comment (rafaelks) [reply Rocket.Chat.xcodeproj/project.pbxproj]: Can you remove this change then?

Comment (Sameesunkaria) [reply Rocket.Chat.xcodeproj/project.pbxproj]: should be fixed now
